Tasks/Common/SonarQubeHelpers/SonarQubeHelper.ps1

```diff
@@ -12,22 +12,30 @@ function EscapeArg
 
 
 # Set a variable in a property bag that is accessible by all steps
-# To retrieve the variable use $val = Get-Variable $distributedTaskContext "varName"
+# To retrieve the variable use GetTaskContextVariable
+# The variable will be available in both the current and the subsequent tasks
 function SetTaskContextVariable
 {
     param([string][ValidateNotNullOrEmpty()]$varName, 
           [string]$varValue)
         
-    #[Environment]::SetEnvironmentVariable($varName, $varValue)
-    Write-Host "##vso[task.setvariable variable=$varName;]$varValue"
+    [Environment]::SetEnvironmentVariable($varName, $varValue)
```

Comment:
This is a slightly non-related change. In the new coreclr agent, the task variables commands no longer work inside the same task. So I'm using env variables to workaround that.